test/DynamoCoreWpfTests/ViewExtensions/NotificationsExtensionTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Dynamo.Utilities;
+using Dynamo.Notifications.View;
+using NUnit.Framework;
+using System;
+using System.IO;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+
+namespace DynamoCoreWpfTests.ViewExtensions
+{
+    public class NotificationsExtensionTests : DynamoTestUIBase
+    {
+        [Test]
+        public void PressNotificationButtonAndShowPopup()
+        {
+            var shortcutBar = this.View.ShortcutBar;
+            var notificationsButton = (Button)shortcutBar.FindName("notificationsButton");
+            notificationsButton.RaiseEvent(new RoutedEventArgs(ButtonBase.ClickEvent));
+
+            NotificationUI notificationUI = PresentationSource.CurrentSources.OfType<System.Windows.Interop.HwndSource>()
+                                        .Select(h => h.RootVisual)
+                                        .OfType<FrameworkElement>()
+                                        .Select(f => f.Parent)
+                                        .OfType<NotificationUI>()
+                                        .FirstOrDefault(p => p.IsOpen);
+
+            Assert.NotNull(notificationUI);
+            var webView = notificationUI.FindName("webView");
+            Assert.NotNull(webView);
+        }
+
+        [Test]
+        public void ValidateNotificationsUIEmbeededFiles()
```

Comment:
`ValidateNotificationsUIEmbeededFiles` spell error, update to - > `ValidateNotificationsUIEmbededFiles`